content/en/platform/corda/5.2/deploying-operating/deployment/deploying/_index.md

```diff
@@ -414,108 +411,399 @@ The following configuration is required for the Corda databases:
+        valueFrom:
+          secretKeyRef:
+            key: <POSTGRESQL_CRYPTO_WORKER_PASSWORD_SECRET_KEY>
+            name: <POSTGRESQL_CRYPTO_WORKER_PASSWORD_SECRET_NAME>
+  db:
+    config:
+      username:
+        valueFrom:
+          secretKeyRef:
+            key: <POSTGRESQL_DB_WORKER_USERNAME_SECRET_KEY>
+            name: <POSTGRESQL_DB_WORKER_USERNAME_SECRET_NAME>
+      password:
+        valueFrom:
+          secretKeyRef:
+            key: <POSTGRESQL_DB_WORKER_PASSWORD_SECRET_KEY>
+            name: <POSTGRESQL_DB_WORKER_PASSWORD_SECRET_NAME>
+  persistence:
+    config:
+      username:
+        valueFrom:
+          secretKeyRef:
+            key: <POSTGRESQL_PERSISTENCE_WORKER_USERNAME_SECRET_KEY>
+            name: <POSTGRESQL_PERSISTENCE_WORKER_USERNAME_SECRET_NAME>
+      password:
+        valueFrom:
+          secretKeyRef:
+            key: <POSTGRESQL_PERSISTENCE_WORKER_PASSWORD_SECRET_KEY>
+            name: <POSTGRESQL_PERSISTENCE_WORKER_PASSWORD_SECRET_NAME>
+  tokenSelection:
+    config:
+      username:
+        valueFrom:
+          secretKeyRef:
+            key: <POSTGRESQL_TOKEN_SELECTION_WORKER_USERNAME_SECRET_KEY>
+            name: <POSTGRESQL_TOKEN_SELECTION_WORKER_USERNAME_SECRET_NAME>
+      password:
+        valueFrom:
+          secretKeyRef:
+            key: <POSTGRESQL_TOKEN_SELECTION_WORKER_PASSWORD_SECRET_KEY>
+            name: <POSTGRESQL_TOKEN_SELECTION_WORKER_PASSWORD_SECRET_NAME>
+  uniqueness:
+    config:
+      username:
+        valueFrom:
+          secretKeyRef:
+            key: <POSTGRESQL_UNIQUENESS_WORKER_USERNAME_SECRET_KEY>
+            name: <POSTGRESQL_UNIQUENESS_WORKER_USERNAME_SECRET_NAME>
+      password:
+        valueFrom:
+          secretKeyRef:
+            key: <POSTGRESQL_UNIQUENESS_WORKER_PASSWORD_SECRET_KEY>
+            name: <POSTGRESQL_UNIQUENESS_WORKER_PASSWORD_SECRET_NAME>
+```
+
+Note that, as elsewhere, credentials can be specified directly as part of the overrides using `value` instead of `valueFrom`:
+
+```yaml
+username:
+  value: <USERNAME>
+password:
+  value: <USERNAME>
+```
+
+{{< note >}}
+Although the password can be specified directly as a Helm override, R3 does not recommend this for security reasons.
+Similarly, although all Corda workers can share the same set of credentials, R3 recommends you create a Kubernetes Secret per Corda worker containing the credentials.
+{{< /note >}}
+
+If required, the connection pool used when interacting with the database can also be independently configured at the `config` level within each worker:
+
+```yaml
+config:
+  connectionPool:
+    minSize: <MIN_CONNECTION_POOL_SIZE>
+    maxSize: <MAX_CONNECTION_POOL_SIZE>
+    maxLifetimeSeconds: <MAX_CONNECTION_POOL_LIFE_TIME_SECONDS>
+    idleTimeoutSeconds: <MAX_CONNECTION_POOL_IDLE_TIMEOUT_SECONDS>
+    keepAliveTimeSeconds: <MAX_CONNECTION_POOL_KEEP_ALIVE_TIME_SECONDS>
+    validationTimeoutSeconds: <MAX_CONNECTION_POOL_VALIDATION_TIME_OUT_SECONDS>
```

Comment:
We good if, as in the state manager case, this was given as an example under a worker (given we have a top-level `config` field).

Reply:
👍